utils/mail.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from requests.exceptions import ConnectionError  # pylint: disable=import-error
```

Comment:
Normally if you write that on the first line of the module, it will extend to the whole module (but I agree it might be better to explicitly limit this disabling of pylint to single lines :))

Reply:
I also favor a per-line mocking